DqtApi/src/DqtApi/Services/TrnGenerationApi/ITrnGenerationApiClient.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DqtApi.Services.TrnGenerationApi
+{
+    using System.Threading.Tasks;
+
+    public interface ITrnGenerationApiClient
+    {
+        Task<string> GenerateTrnAsync();
```

Comment:
We don't tend to add an `Async` suffix to method names
```suggestion
        Task<string> GenerateTrn();
```

Reply:
Amended